charts/vald/templates/rebalancer/jobconfigmap.yaml

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (C) 2019-2021 vdaas.org vald team <vald@vdaas.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+{{- if .Values.rebalancer.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.rebalancer.rebalancer.rebalance_job_configmap_name }}
+  labels:
+    app.kubernetes.io/name: {{ include "vald.name" . }}
+    helm.sh/chart: {{ include "vald.chart" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.Version }}
+    app.kubernetes.io/component: rebalancer
+data:
+  job.tpl: |
+    ---
+    apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: {{ .Values.rebalancer.rebalancer.rebalance_job_name }}
+      namespace: vald
```

Comment:
https://github.com/vdaas/vald/pull/836#discussion_r630881748
From rinx: 
Is it okay to be specified?